src/components/Channel/Channel.tsx

```diff
@@ -112,6 +112,8 @@ type ChannelPropsForwardedToComponentContext<
   BaseImage?: ComponentContextValue<StreamChatGenerics>['BaseImage'];
   /** Custom UI component to display the slow mode cooldown timer, defaults to and accepts same props as: [CooldownTimer](https://github.com/GetStream/stream-chat-react/blob/master/src/components/MessageInput/CooldownTimer.tsx) */
   CooldownTimer?: ComponentContextValue<StreamChatGenerics>['CooldownTimer'];
+  /** Custom UI component to render set of buttons to be displayed in the MessageActionsBox, defaults to and accepts same props as: [CustomMessageActionsList](https://github.com/GetStream/stream-chat-react/blob/master/src/components/MessageActions/CustomMessageActionsList.tsx) */
+  CustomMessageActionsList?: ComponentContextValue<StreamChatGenerics>['CustomMessageActionsList'];
```

Comment:
@arnautov-anton I believe you are not forwarding the prop to ComponentContext. Sorry if I missed something.

Reply:
And you'd be correct.